README.md

```diff
@@ -51,6 +64,12 @@ A coding guide line for React + Redux project @WORD UP
   * Use [normalizr](https://github.com/paularmstrong/normalizr) if needed.
   * [Normalizing State Shape](https://redux.js.org/recipes/structuring-reducers/normalizing-state-shape)
 
+#### Combine `set` actions
+  * Instead of using `state.set('key1', value1).set('key2', 'value2')`, use `withMutations` to wrap multiple actions into one update. Since Immutable.js will perform re-arranging in all the processes and retain all intermediate states.
```

Comment:
I think we should give an actual example here of what it should look like

Reply:
good idea, added in 1ffe8dc